res/commands.json

```diff
@@ -270,14 +270,40 @@
 		"usage": "compensate <amount>",
 		"admin": true
 	},
+	"EVENT": {
+		"brief": "Retrieve information about the running event(s).",
+		"usage": "event",
+		"admin": false
+	},
+	"EVENT_LAUNCH": {
+		"brief": "Starts an event.",
+		"usage": "launch <event-name> <expiry-time-in-hours>",
+		"admin": true
+	},
+	"EVENT_END": {
+		"brief": "Stops the running event manually.",
+		"usage": "endevent",
+		"admin": true
+	},
+	"EVENT_GIVE_NEW_ACCESS": {
+		"brief": "Gives event launching and ending ability to the people with the mentioned role(s).",
+		"usage": "give-new-event-access <role-mention(s)>",
+		"admin": true
+	},
+	"EVENT_TAKE_NEW_ACCESS": {
+		"brief": "Removes event launching and ending ability from the provided role(s) that previously had event launching and ending abilities.",
+		"usage": "take-new-event-access <role-mention(s)>",
+		"admin": true
+	},
 	"FUN_MEME": {
 		"brief": "Shows memes from reddit.",
 		"usage": "meme",
 		"admin": false
 	},
 	"FUN_ROLL": {
 		"brief": "Gets a random number.",
-		"usage": "roll <min> <max>",
+		"usage": "roll <min> <max> or !roll <max>",
```

Comment:
I think these should be on separate lines, sort of like how `!help top` does it.